multigeiger/speaker.cpp

```diff
@@ -137,9 +140,9 @@ void IRAM_ATTR tick(bool high) {
     sequence[5] = 0;
     sequence[6] = led_tick ? (high ? 0 : -1) : -1;
     sequence[7] = 0;  // END
-  } else
-    sequence = NULL;
+  }
```

Comment:
if you want to do it that way, you can also just keep the assignment in the else-branch.

Reply:
alright, done.